shared/src/db/database.ts

```diff
@@ -181,20 +182,27 @@ class Database extends Dexie {
     /**
      * Get all IndexedDB documents of a certain type as Vue Ref
      * @param initialValue - The initial value of the ref while waiting for the query to complete
-     * @param tagType - Optional: The tag type to filter by (only used for tags)
+     * @param postOrTagType - Optional: The tag type to filter by (only used for tags)
```

Comment:
The JSDoc descrition does not mention posts but only tags. It should mention both with the change to `postOrTagType`